### PR TITLE
updated ethsafari event to 2023

### PR DESCRIPTION
**Pull request by Barukimang**
## Description

Hi guys!

Am a Core-contributor for ETHSafari. I just updated the dates and description to the event that we are planning in Kenya for this year from 18 - 24 September.

Let me know if you have any questions.

Thanks!

### CODE REVIEW

File: src/data/community-events.json [hunk 1]
```diff
@@ -589,9 +589,9 @@
     "to": "https://ethsafari.xyz/",
     "sponsor": null,
     "location": "Nairobi & Kilifi, Kenya",
-    "description": "Welcome to the latest greatest ETH event happening in Africa!",
-    "startDate": "2022-09-18",
-    "endDate": "2022-09-25"
+    "description": "Welcome to the latest greatest ETH event happening in Africa! Join the BlockTrain from Nairobi to celebrate an ETH-festival held underneath ancient Boabab trees in Kilifi.",
```
Comment (minimalsm) [inline src/data/community-events.json:592]: BlockTrain? Is this a typo @Barukimang?

Comment (minimalsm) [inline src/data/community-events.json:592]:
```suggestion
    "description": "Welcome to the latest greatest Ethereum event happening in Africa! Join the BlockTrain from Nairobi to celebrate an ETH-festival held underneath ancient Boabab trees in Kilifi.",
```

Comment (Barukimang) [reply src/data/community-events.json:592]: Hi @minimalsm No it's an actual train which attendees can take from Nairobi to Kilifi. We call it the BlockTrain for the occasion :) On the train ppl can hack and just meet and bond with other attendees. It's a nice ride through national parks with lots of elephants

Comment (minimalsm) [reply src/data/community-events.json]: Sounds wonderful. Thanks for the quick reply.

**Review by minimalsm — APPROVED** [review]
LGTM :)

**Pull request merged**